### PR TITLE
e2e for Ocrvs 9783

### PR DESCRIPTION
**Pull request by jamil314**
## Description

Clearly describe what has been changed. Include relevant context or background.
Explain how the issue was fixed (if applicable) and the root cause.

Link this pull request to the GitHub issue (and optionally name the branch `ocrvs-<issue #>`)

## Checklist

- [ ] I have linked the correct Github issue under "Development"
- [ ] I have tested the changes locally, and written appropriate tests
- [ ] I have tested beyond the happy path (e.g. edge cases, failure paths)
- [ ] I have updated the changelog with this change (if applicable)
- [ ] I have updated the GitHub issue status accordingly

### CODE REVIEW

File: e2e/testcases/v2-application/workqueue-flow-2.spec.ts [hunk 1]
```diff
@@ -0,0 +1,317 @@
...
+      await loginToV2(page, CREDENTIALS.LOCAL_REGISTRAR)
+
+      await assertRecordInWorkqueue({
+        page,
+        name: formatName(declaration.child.name),
+        workqueues: [
+          { title: 'Assigned to you', exists: false },
+          //   { title: 'Recent', exists: false }, // https://github.com/opencrvs/opencrvs-core/issues/9785
+          { title: 'Notifications', exists: true },
+          { title: 'Ready for review', exists: true },
```
Comment (Zangetsu101) [inline e2e/testcases/v2-application/workqueue-flow-2.spec.ts]: Should the record show up in both of these workqueues?

**Review by Zangetsu101 — CHANGES_REQUESTED** [review]

File: e2e/testcases/v2-application/workqueue-flow-2.spec.ts (hunk 1 above)
Comment (jamil314) [reply e2e/testcases/v2-application/workqueue-flow-2.spec.ts]: No, updated. Thanks!

**Review by Zangetsu101 — APPROVED** [review]
